src/scss/views/widgets/_pill.scss

```diff
@@ -0,0 +1,59 @@
+.pill {
+  display: flex;
+  align-items: center;
+  width: fit-content;
+  height: 32px;
+  border-radius: $border-radius-large;
+  font-size: 14px;
+
+  .pill-title {
+    font-weight: $font-weight-semibold;
+    color: $color-white;
+    margin: 0px 16px;
```

Comment:
ok for now, but i'm gonna address this with you and mike, we have wayy to many hardcoded sizes, when we should be using variables like `base-spacing`, `base-spacing-quad` or wtv, or making them if we don't have em

IIRC mark is designing everything based off of `4px`, so we should avoid hardcoding these spacings everywhere.